### PR TITLE
Genesis block is now a json file

### PR DESCRIPTION
**Pull request by mvines**
The json file is interpreted explicitly by the bank, so as a result genesis entries no longer show up in the ledger.

Example genesis.json, which sits in a `ledger/` directory along side the `rocksdb/` subdirectory:
```json
cat config/ledger/genesis.json 
{  
   "bootstrap_leader_id":[ ... ],
   "bootstrap_leader_tokens":1000000,
   "mint_id":[ ... ],
   "tokens":1000000000
}
```

TODO:
* [x] Fix all the tests

### CODE REVIEW

File: src/fullnode.rs [hunk 1]
```diff
@@ -187,16 +219,16 @@ impl Fullnode {
 
     /// Create a fullnode instance acting as a leader or validator.
     #[allow(clippy::too_many_arguments)]
-    pub fn new_with_bank(
+    pub fn new_with_bank_and_db_leader(
```
Comment (rob-solana) [inline src/fullnode.rs]: db_leader?  db_ledger?

File: ledger-tool/src/main.rs [hunk 2]
```diff
@@ -103,26 +112,7 @@ fn main() {
             stdout().write_all(b"\n]}\n").expect("close array");
         }
         ("verify", _) => {
-            const NUM_GENESIS_ENTRIES: usize = 3;
```
Comment (rob-solana) [inline ledger-tool/src/main.rs]: nice

File: src/bank.rs [hunk 3]
```diff
@@ -732,19 +712,14 @@ impl Bank {
...
-    ) -> Result<(u64, Hash)>
+    pub fn process_ledger<I>(&self, entries: I) -> Result<(u64, Hash)>
     where
         I: IntoIterator<Item = Entry>,
     {
         // these magic numbers are from genesis of the mint, could pull them
         //  back out of this loop.
-        let mut entry_height = entry_height;
-        let mut last_id = start_hash;
+        let mut entry_height = 0;
```
Comment (rob-solana) [inline src/bank.rs]: nice

**Comment by mvines** [comment]
🛎 It's ready if you wanna take a look @garious

**Pull request merged**